src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2017 Expedia Inc.
```

Comment:
Inception year of 2016 not being picked up properly, you need to get plugin to regenerate all headers